File: URI/1 - INICIANTE/C/1176 - FibonacciEmVetor.c
#include <stdio.h>

int main(){
	int x, num;
	unsigned long long int vetor[60], n1, n2, n3;
	n1 = 0;
	n2 = 1;
	vetor[0] = 0;
	for(x = 1; x < 60; x++){
			vetor[x] = n2;
			n3 = n2;
			n2 = n1 + n2;
			n1 = n3;
	}	
	scanf(" %d", &num);
	while(num > 0){
		scanf(" %d", &x);
		printf("Fib(%d) = %lld\n", x, vetor[x]);
		num--;
	}	
	return 0;
}

File: URI/1 - INICIANTE/Python/1176 - FibonacciEmVetor.py
vetor = []
n1 = 0
n2 = 1
vetor.append(0)
for x in range(0, 60):
    vetor.append(n2)
    n3 = n2
    n2 = n1 + n2
    n1 = n3
num = int(input())
while num > 0:
    x = int(input())
    print('Fib({}) = {}'.format(x, vetor[x]))
    num -= 1